lib/core/diagnostics.js

```diff
@@ -0,0 +1,214 @@
+let channels
```

Comment:
I'm pretty sure the try catch can be removed since it was only needed in node versions that didn't support diagnostics channel

Reply:
Yeah, had the same thought but wasn't sure about that, so put it as a safe net. >18 shouldn't be a problem.

I'll remove it then